src/PluginsSystem/Tools/Microsoft.Performance.Toolkit.Plugins.Cli.Tests/PluginManifestSerializationTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using AutoFixture;
+using Microsoft.Performance.Toolkit.Plugins.Cli.Manifest;
+using Microsoft.Performance.Toolkit.Plugins.Core.Serialization;
+using Fixture = AutoFixture.Fixture;
+
+namespace Microsoft.Performance.Toolkit.Plugins.Cli.Tests;
+
+[TestClass]
+public sealed class PluginManifestSerializationTest
+{
+    private Fixture? fixture = null;
+
+    [TestInitialize]
+    public void Setup()
+    {
+        this.fixture = new Fixture();
+    }
+
+    [TestMethod]
+    public void PluginManifest_Serialization_RoundTrip()
+    {
+        ISerializer<PluginManifest> manifestSerializer = SerializationUtils.GetJsonSerializer<PluginManifest>(Program.PluginManifestSerializerDefaultOptions);
```

Comment:
Same comment here about duplicating test code. Is there a way we can re-use the logic between the two files?